oak_functions/examples/key_value_lookup/config.toml

```diff
@@ -1,6 +1,7 @@
 lookup_data_url = "https://storage.googleapis.com/oak_lookup_data/lookup_data_weather"
 lookup_data_download_period = "10m"
 worker_threads = 4
+tee_certificate = "./examples/certs/local/ca.pem"
```

Comment:
Does this need to be added to some docker file, or am I missing how it is going to work?

Reply:
Good point, I've created #2141 to fix the Cloud Run deployment. 
I think we need something similar to the following: https://github.com/project-oak/oak/blob/c6c6e4a7974228bf05e5ebb4f5151fde41b1d7f1/examples/aggregator/scripts/deploy#L41-L44